src/Network/Wai/Middleware/Servant/Errors.hs

```diff
@@ -57,8 +58,8 @@ module Network.Wai.Middleware.Servant.Errors
   , encodeAsPlainText
   )where
 
-import Prelude.Compat
-import Data.Aeson (Value (..), encode)
+import "base-compat" Prelude.Compat
```

Comment:
This change is not merge blocking but I am curious whether it's necessary?

Reply:
Ghci yelled at me without this 🤷🏼‍♂️ there might be some conflicting module in the package dependencies